src/Sys/InspectSeoHandler.php

```diff
@@ -0,0 +1,177 @@
+<?php
+declare(strict_types=1);
+
+namespace Triniti\Sys;
+
+use Defuse\Crypto\Crypto;
+use Gdbots\Ncr\Exception\NodeNotFound;
+use Gdbots\Ncr\Ncr;
+use Gdbots\Pbj\Message;
+use Gdbots\Pbjx\CommandHandler;
+use Gdbots\Pbjx\Pbjx;
+use Gdbots\Schemas\Pbjx\StreamId;
+use Gdbots\UriTemplate\UriTemplateService;
+use Google\Service\SearchConsole\InspectUrlIndexResponse;
+use Psr\Log\LoggerInterface;
+use Psr\Log\NullLogger;
+use Triniti\Schemas\Sys\Event\SeoInspectedV1;
+use Defuse\Crypto\Key;
+
+
+class InspectSeoHandler implements CommandHandler
+{
+    public static function handlesCuries(): array
+    {
+        return [
+            'triniti:sys:command:inspect-seo',
+        ];
+    }
+
+    public function __construct(
+        private readonly Ncr             $ncr,
+        private readonly Key             $key,
+        private readonly Flags           $flags,
+        private readonly Pbjx            $pbjx,
```

Comment:
handleCommand has pbjx already so you probably can use that instead of passing in the constructor